enterprise/api_client/sso_orchestrator.py

```diff
@@ -102,7 +103,7 @@ def _post(self, url, data=None):
                 f"Failed to make SSO Orchestrator API request: {response.status_code}",
                 response=response,
             )
-        return response.status_code
+        return response
```

Comment:
perhaps you'd rather `return response.json()` here? most of our API clients return a `dict` of the data versus a raw response object?